v3/src/components/app.tsx

```diff
@@ -117,8 +115,10 @@ export const App = observer(function App() {
       const sample = sampleData.find(name => urlParams.sample === name.toLowerCase())
       if (sample) {
         importSample(sample, handleImportDataSet)
-      } else {
+      }
+      if (urlParams.dashboard !== undefined) {
         createNewStarterDataset()
+        addDefaultComponents()
       }
```

Comment:
Do we need the call to `createNewStarterDataSet` if we're not importing a sample? I was expecting something more like:
```
      if (sample) {
        importSample(sample, handleImportDataSet)
      } else {
        createNewStarterDataset()
      }
      if (urlParams.dashboard !== undefined) {
        addDefaultComponents()
      }
```